content/blog/2021-02-01-tidynhl-analyse/index.fr.md

```diff
@@ -0,0 +1,711 @@
+---
+title: "Repêchage de la LNH: est-ce vraiment une science inexacte?"
+slug: "tidy-nhl-draft-analysis"
+author: "Stéphane Caron"
+description: ""
+date: "2021-02-12"
+categories: ["Analytique"]
+type: post
+tags: 
+  - R
+  - Analytique de sports
+output:
+  html_document:
+    keep_md: yes
+featured: "hockey-draft-post.jpg"
+featuredpath: "img/headers/"
+reading_time: 10
+---
+
+
+
+**On entend souvent dire que le repêchage de la LNH, tout comme c'est le cas
+dans d'autres sports professionnels, est une science inexacte. J'imagine que cette
+expression fait référence au fait que si l'on doit repêcher un joueur $x$ à un moment 
+$t$ donné, il n'y a aucune certitude que ce joueur est le bon choix. Mais disons 
+qu'on prend le temps de regarder plusieurs choix, sur plusieurs années, est-ce toujours le 
+cas? Est-ce que le repêchage devient en quelque sorte une science plus "exacte" lorsque l'échantillon de joueurs repêchés devient plus grand?**
+
+Dans cet article, je tenterai d'éclairer cette question en analysant les récents
+repêchages de la LNH. Pour se faire, j'utiliserai différentes fonctionnalités du
+_package_ <svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 581 512"><path d="M581 226.6C581 119.1 450.9 32 290.5 32S0 119.1 0 226.6C0 322.4 103.3 402 239.4 418.1V480h99.1v-61.5c24.3-2.7 47.6-7.4 69.4-13.9L448 480h112l-67.4-113.7c54.5-35.4 88.4-84.9 88.4-139.7zm-466.8 14.5c0-73.5 98.9-133 220.8-133s211.9 40.7 211.9 133c0 50.1-26.5 85-70.3 106.4-2.4-1.6-4.7-2.9-6.4-3.7-10.2-5.2-27.8-10.5-27.8-10.5s86.6-6.4 86.6-92.7-90.6-87.9-90.6-87.9h-199V361c-74.1-21.5-125.2-67.1-125.2-119.9zm225.1 38.3v-55.6c57.8 0 87.8-6.8 87.8 27.3 0 36.5-38.2 28.3-87.8 28.3zm-.9 72.5H365c10.8 0 18.9 11.7 24 19.2-16.1 1.9-33 2.8-50.6 2.9v-22.1z"/></svg>
```

Comment:
```suggestion
repêchages de la LNH. Pour se faire, j'utiliserai différentes fonctionnalités de la 
_librairie_ <svg style="height:0.8em;top:.04em;position:relative;fill:steelblue;" viewBox="0 0 581 512"><path d="M581 226.6C581 119.1 450.9 32 290.5 32S0 119.1 0 226.6C0 322.4 103.3 402 239.4 418.1V480h99.1v-61.5c24.3-2.7 47.6-7.4 69.4-13.9L448 480h112l-67.4-113.7c54.5-35.4 88.4-84.9 88.4-139.7zm-466.8 14.5c0-73.5 98.9-133 220.8-133s211.9 40.7 211.9 133c0 50.1-26.5 85-70.3 106.4-2.4-1.6-4.7-2.9-6.4-3.7-10.2-5.2-27.8-10.5-27.8-10.5s86.6-6.4 86.6-92.7-90.6-87.9-90.6-87.9h-199V361c-74.1-21.5-125.2-67.1-125.2-119.9zm225.1 38.3v-55.6c57.8 0 87.8-6.8 87.8 27.3 0 36.5-38.2 28.3-87.8 28.3zm-.9 72.5H365c10.8 0 18.9 11.7 24 19.2-16.1 1.9-33 2.8-50.6 2.9v-22.1z"/></svg>
```

Reply:
IMO librairie plutôt que _package_

Reply:
Dans l'univers de **R** j'ai l'impression le terme librairie est pas vraiment utilisé .. plutôt package. Je l'ai mis en italique au moins. On verra ce que le maitre Jedi R @jplecavalier en pense ... Sinon on demandera au grand maitre Jedi (Vincent) hahah

Reply:
Le terme français reste librairie, c'est juste de la loi 101 ici.

Reply:
@davebulaval t'étais pas au R à QC 2017 ? Vigou a dit paquetage!

Reply:
True that

Reply:
A library is a collection of many packages.
Une librairie est un collection de plusieurs paquetages.

Ceci étant dit, j'utilise *package* parce que je n'aime pas le mot paquetage. C'est honnêtement rien de plus qu'un choix personnel.